pymeasure/instruments/agilent/agilentB1500.py

```diff
@@ -1625,7 +1922,7 @@ class ADCType(CustomIntEnum):
 
     HSADC = (0,)  #: High-speed ADC
     HRADC = (1,)  #: High-resolution ADC
-    HSADC_PULSED = (2,)  #: High-resolution ADC for pulsed measurements
+    HSADC_PULSED = (2,)  #: High-speed ADC for pulsed measurements
```

Comment:
That still continues to work in all used places, as int is not tuple of one member?